link/kprobe.go

```diff
@@ -294,3 +303,37 @@ func kprobePrefix(ret bool) string {
 	}
 	return "p"
 }
+
+// determineRetprobeBit reads a Performance Monitoring Unit's retprobe bit
+// from /sys/bus/event_source/devices/<pmu>/format/retprobe.
+func determineRetprobeBit(pmu string) (uint64, error) {
+	parseFunc := func(pmu string) (uint64, error) {
+		p := filepath.Join("/sys/bus/event_source/devices/", pmu, "/format/retprobe")
+
+		data, err := ioutil.ReadFile(p)
+		if err != nil {
+			return 0, err
+		}
+
+		var rp uint64
```

Comment:
Could a uint8 be used here instead?

Reply:
Yes, but later we will use an uint64 anyway

Reply:
Cool, thanks!